dds/DCPS/RTPS/Sedp.cpp

```diff
@@ -1138,20 +1132,6 @@ Sedp::Task::svc_i(const ParticipantData_t* ppdata)
   }
 #endif
 
-  // Write durable data
```

Comment:
Is there any problem with the `defer_match_endpoints_` happening before `write_durable_*_data`?

Reply:
Initial testing seems to indicate we can remove both defer_match_endpoints and first_acknowledged_by_reader. Pushing the removals as individual commits for each so hopefully it will be easy to revert if we discover issues later.